tests/test_schema.py

```diff
@@ -4162,6 +4162,29 @@ def test_schema_migrations_equivalence_rename_refs_06(self):
             alias Alias3 := Remark { command := .remark ++ "!" };
         """])
 
+    def test_schema_migrations_equivalence_rename_abs_constraint_01(self):
```

Comment:
Please add an (xfailing?) test using the `assert_describe_migration` framework (see `test_edgeql_data_migraiton`).

Reply:
Looks like we already have the xfailed test we want `test_edgeql_migration_describe_constraint_01`, so I just updated its description